util/unittests/test_process.py

```diff
@@ -132,7 +132,7 @@ def test02_getpid_listeningonport(self):
                                                 self.ssh_client),
                          'Testing when port is not owned. Remote')
 
-    def test03_getpid_listeningonport(self):
+    def test_getpid_listeningonport(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_getpid_listeningonport' from line 119

<!-- Comment by [SideCI](https://sideci.com) -->